pkg/analysis_server/lib/src/services/correction/fix_internal.dart

```diff
@@ -20,9 +20,9 @@ import 'package:analysis_server/src/services/correction/levenshtein.dart';
 import 'package:analysis_server/src/services/correction/name_suggestion.dart';
 import 'package:analysis_server/src/services/correction/namespace.dart';
 import 'package:analysis_server/src/services/correction/source_buffer.dart';
+import 'package:analysis_server/src/services/correction/source_range.dart';
```

Comment:
This library is already imported (on the line below) with a prefix. Seems like we should have one or the other, but not both.